### PR TITLE
Add april 2024

### PR DESCRIPTION
**Pull request by Bestem0r**

### CODE REVIEW

**Review by ivarnakken — APPROVED** [review]

File: _posts/2024-05-07-mai.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Utviklarbloggen har fått nytt namn
+
+Ut med det gamle, og inn med det nye! 🎉 Utviklerbloggen har fått nytt namn, og heiter frå no av **Utviklarbloggen**! 🤩 Dette er ein stor milepæl for bloggen, og eit viktig steg i riktig retning for Abakus sin språkprofil. Me håper at det nye namnet fell i smak, og at de framleis vil følgje med på kva Webkom driv med! 🥳
+
+![Utviklarbloggen](/images/posts/2024-05-07-new-name.png)
+
+<br>
+## Nye tips i interesseskjema
+
+For verksemder interesserte i Abakus, er det ikkje alltid like enkelt å vite kva ein "Bedriftspresentasjon" eller "Fukostføredrag" eigentleg dreier seg om. Me har derfor lagt til nokre **verktøytips** i interesseskjemaet, som forklarer kva dei ulike arrangementa går ut på. Dette gjer det enklare for verksemder å velje kva type arrangement dei ynskjer å arrangere! 👨‍🏫
```
Comment (ollfkaih) [inline _posts/2024-05-07-mai.md:19]:
```suggestion
For verksemder interesserte i Abakus, er det ikkje alltid like enkelt å vite kva ein "Bedriftspresentasjon" eller "Frokostforedrag" eigentleg dreier seg om. Me har derfor lagt til nokre **verktøytips** i interesseskjemaet, som forklarer kva dei ulike arrangementa går ut på. Dette gjer det enklare for verksemder å velje kva type arrangement dei ynskjer å arrangere! 👨‍🏫
```

**Review by ollfkaih — APPROVED** [review]
Just a minor linguistic error, god otherwise

File: _posts/2024-05-07-mai.md (hunk 1 above)
Comment (Bestem0r) [reply _posts/2024-05-07-mai.md:19]: https://ordbokene.no/bm,nn/frukost

Comment (Bestem0r) [reply _posts/2024-05-07-mai.md:19]: Or should it be the "official" name since it's an Abakus concept?

Comment (ollfkaih) [reply _posts/2024-05-07-mai.md:19]: https://ordbokene.no/bm,nn/fukost

**Pull request merged**